pkg/config/config.go

```diff
@@ -54,6 +55,21 @@ type Xen struct {
 	XenBridge  string `yaml:"xen_bridge"`
 }
 
+type Openstack struct {
+	Name         string `yaml:"name"`
+	OSUserName   string `yaml:"os_username"`
```

Comment:
I suggest you don't use `OS` prefixes here. These properties are part of OpenStack configuration.